Sources/AirbnbSwiftFormatTool/AirbnbSwiftFormatTool.swift

```diff
@@ -83,6 +86,10 @@ struct AirbnbSwiftFormatTool: ParsableCommand {
       arguments += ["--lint"]
     }
 
+    if let swiftVersion = swiftVersion {
+      arguments += ["--swiftversion", swiftVersion]
```

Comment:
Will this take precedence over https://github.com/airbnb/swift/blob/1bf961396cc9c690db37936bfbdc8b5f29e844af/Sources/AirbnbSwiftFormatTool/airbnb.swiftformat#L2?

Reply:
Yes, it does